DNN Platform/Library/Entities/Portals/PortalStylesController.cs

```diff
@@ -0,0 +1,128 @@
+﻿#region Copyright
+
+// 
+// DotNetNuke® - http://www.dotnetnuke.com
+// Copyright (c) 2002-2018
+// by DotNetNuke Corporation
+// 
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated 
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation 
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and 
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// 
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions 
+// of the Software.
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED 
+// TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL 
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF 
+// CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER 
+// DEALINGS IN THE SOFTWARE.
+
+#endregion
+
+
+#region usings
+using DotNetNuke.Collections;
+using DotNetNuke.ComponentModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+#endregion
+
+namespace DotNetNuke.Entities.Portals
+{
+    class PortalStylesController : IPortalStylesController
```

Comment:
2 items on this.  1.) Is it really intended to be "Internal" if so, lets decorate it as that.  2.) Can I get an XML comment on it too ;)

Reply:
I do not understand what you mean by Internal, this is not right now internal, should it become? I would see that modules could need to obtain those values on the server side for some purposes so I think it should be public.

Reply:
I am a bit confused, maybe we can discuss this next time we meet...

Reply:
Currently, there is no access modifier on the class.  Therefore, it is by default **public**.